workshops/models.py

```diff
@@ -18,6 +18,7 @@
 STR_SHORT   =  10         # length of short strings
 STR_MED     =  40         # length of medium strings
 STR_LONG    = 100         # length of long strings
+STR_LONGEST = 255  # length of the longest strings
```

Comment:
Yes, but my intention was to add a variable substituting `max_length=255`, which was used in many places :)